util.go

```diff
@@ -24,7 +23,7 @@ func Errc(e error, message ...string) {
 		if message != nil {
 			log.Println(message)
 		}
-		log.Fatal(e)
+		panic(e)
```

Comment:
panic() dumps unnecessary debug info and does not close file handlers properly. use log.Fatal, since that calls exit(1)

Reply:
Using `panic()` is intentional behavior, its how errors not handled by us should be handled, the debug information is required for further debugging when someone else comes to this crash.

Other than that, the panic screen looks nice.

Reply:
> does not close file handlers properly. use log.Fatal, since that calls exit(1)

Doesn't the go runtime handle this ...?

Reply:
OK makes sense. Errc should never(?) trigger anyway under normal conditions